src/msw/dc.cpp

```diff
@@ -2624,11 +2624,11 @@ static bool AlphaBlt(wxMSWDCImpl* dcDst,
             if ( data )
             {
                 wxAlphaPixelData::Iterator p(data);
-                for ( int y = 0; y < data.GetHeight(); y++ )
+                for ( int yy = 0; yy < data.GetHeight(); yy++ )
```

Comment:
Maybe this would be more clear as `old_x` and `old_y` (referring to the alpha pixel data "bmpOld" it's iterating on).